lms/djangoapps/instructor/views/legacy.py

```diff
@@ -736,8 +751,20 @@ def get_student_grade_summary_data(
                         else:
                             add_grade(score.section, score.earned)
                 else:
+                    category_cnts = Counter()
+                    progress_summary = grades._progress_summary(student, request, course)
                     for grade_item in gradeset['section_breakdown']:
-                        add_grade(grade_item['label'], grade_item['percent'])
+                        category = grade_item['category']
+                        try:
+                            location = gradeset['totaled_scores'][category][category_cnts[category]].module_id
+                            earned, possible = progress_summary.score_for_module(location)
+                            if get_score_max is True:
+                                add_grade(grade_item['label'], earned, possible=possible)
+                            else:
+                                add_grade(grade_item['label'], grade_item['percent'], possible=1)
+                        except (IndexError, KeyError):
```

Comment:
actually it was @pwilkins work (try, catch). I havent changed it much. I am just loading max points from progress_summery. 

@pwilkins can you explain me why you used this try catch statement?

cc @pdpinch